lib/spack/llnl/util/symlink.py

```diff
@@ -54,7 +53,16 @@ def _win32_junction(path, link):
         path = os.path.join(parent, path)
         path = os.path.abspath(path)
 
-    CreateHardLink(link, path)
+    command = "mklink"
+    default_args = [link, path]
+    if os.path.isdir(path):
+        # try using a junction
+        default_args.insert(0, '/J')
+    else:
+        # try using a hard link
+        default_args.insert(0, '/H')
+
+    Executable(command)(*default_args)
```

Comment:
This appears to use changes from https://github.com/spack/spack/pull/27021 which were later corrected in https://github.com/spack/spack/pull/29711 - was the updated code giving you problems?

Reply:
No, in fact I don't recall touching this file at any point during my coding. Should I get rid of this?

Reply:
Yes